photon-client/src/main/scala/com/linkedin/photon/ml/util/IOUtils.scala

```diff
@@ -222,7 +223,7 @@ object IOUtils {
     val stream = fs.create(outputPath, forceOverwrite)
     val writer = new PrintWriter(
       new BufferedWriter(
-        new OutputStreamWriter(stream)
+        new OutputStreamWriter(stream, UTF_8.name())
```

Comment:
You can pass the `UTF_8` charset directly -- no need to call `name()`.

Reply:
thank you for your suggestion! Works as well